shared/actions/unlock-folders.js

```diff
@@ -74,20 +74,45 @@ export function registerRekeyListener (): (dispatch: Dispatch) => void {
       engine.rpc(params)
     })
 
-    createServer(
-      engine,
-      'keybase.1.rekeyUI.delegateRekeyUI',
-      null,
-      () => ({
-        'keybase.1.rekeyUI.delegateRekeyUI': (params, response) => { },
-        'keybase.1.rekeyUI.refresh': ({sessionID, problemSetDevices}, response) => {
-          console.log('Asked for rekey')
-          dispatch(({type: Constants.newRekeyPopup, payload: {devices: problemSetDevices.devices || [], sessionID}}: NewRekeyPopupAction))
-          uglySessionIDResponseMapper['refresh'] = response
-          response.result()
-        },
-      })
-    )
+    if (__DEV__ && false) {
```

Comment:
this is so you can run ```keybase rekey trigger``` which will force a rekey popup. we can kill this once this is all settled down